ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -230,6 +231,54 @@
     "PageID": "defenceRoute",
     "ShowSummaryChangeOption": "N"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
```

Comment:
As discussed during the review session, please can we try isolating these in a separate file for example DefendantResponseAdmitPartLRspec.json to modularise the flow logic.